### PR TITLE
Add docs about highlighted counter

### PR DESCRIPTION
**Pull request by nickvergessen**
Ref https://github.com/nextcloud/server/pull/9363

### CODE REVIEW

**Comment by nickvergessen** [comment]
Updated with https://github.com/nextcloud/server/pull/9363/commits/9b0f17b248114f6d50d91887037fc9c138b137e2

File: developer_manual/design/navigation.rst [hunk 1]
```diff
@@ -226,7 +226,7 @@ Do not change the alignment of the text. If you're using
 
     <li class="app-navigation-entry-utils-counter">1</li>
 
-The count should be limitted to 999 and turn to 999+ if any higher number is given. If AngularJS is used the following filter can be used to get the correct behavior:
+The counter should be limited to 999 and turn to 999+ if any higher number is given. If AngularJS is used the following filter can be used to get the correct behavior:
```
Comment (MariusBluem) [inline developer_manual/design/navigation.rst]: @nickvergessen 99+ instead of 999+, or am I wrong? https://github.com/nextcloud/spreed/pull/845

Comment (nickvergessen) [reply developer_manual/design/navigation.rst]: No, you read the comments wrong

Comment (MariusBluem) [reply developer_manual/design/navigation.rst]: I see. (At least I think so 😅)

**Comment by MorrisJobke** [comment]
stable13 88244e43 456ec675

**Comment by skjnldsv** [comment]
@MorrisJobke this is a design feature and was not backported

**Comment by MorrisJobke** [comment]
@skjnldsv But the spreed app uses it?! Or was this then ported in the design app itself?

**Comment by nickvergessen** [comment]
Yes, we copied the CSS to the spreed app.

**Comment by MorrisJobke** [comment]
Okay - let me revert the back ports then.

**Comment by MorrisJobke** [comment]
Reverte

**Pull request merged**